pre-docs/inventory_version2.md

```diff
@@ -464,6 +464,67 @@ plugins:
     credentials: ~/alternate_path/credentials
 ```
 
+#### Azure
+
+The Azure plugin supports looking up virtual machines and virtual machine scale sets. It supports several fields:
+
+- `_plugin`: The value of `_plugin` must be `azure`
+- `resource_group`: The resource group to filter by (optional)
+- `scale_set`: The scale set to filter by (optional, requires that `resource_group` also be set)
+- `location`: The location to filter by (optional)
+- `tags`: A list of one or more tags to filter by (optional, tags are `name: value` pairs and are ANDed together)
+
+Accessing Azure resources requires credentials for signing all API requests. Each credential can be set in the inventory config, Bolt config, or as an environment variable, and are searched in this order until a value is found.
+
+- `tenant_id`: The Azure AD tenant ID (optional, defaults to `ENV['AZURE_TENANT_ID']`)
```

Comment:
These don't have to be specified where you _invoke_ the plugin, but you have to have a value for all of them set _somewhere_. Does that mean they're optional or not?

Reply:
The credentials themselves aren't optional, but don't necessarily have to be set in the inventory/config. I can remove the 'optional' part if it makes it clearer.